### PR TITLE
Refactor configuration and adopt black

### LINKED ISSUE #625: Adopt consistent formatting via black
https://github.com/psf/black

To avoid personal reformatting which leads to a messed up git history and unnecessary conflicts in other PRs.

1) Do once a reformat with a different author, s.t. it can be filtered from the history
2) Add a `black --check` to the CI
3) Recommend editor setup (e.g. for Vim: `autocmd BufWritePre *.py execute ':Black'`)

There are tons of guides, e.g. https://www.mattlayman.com/blog/2018/python-code-black/

### PR DESCRIPTION
**Pull request by anukul**
Closes #625

### CODE REVIEW

**Comment by anukul** [comment]
Should we add a pre-commit hook that runs `black`?

File: setup.py [hunk 1]
```diff
@@ -39,10 +39,7 @@
     ],
     extras_require={
         'stylecheck': [
-            'flake8==3.5.0',
-            'flake8-quotes==1.0.0',
-            'flake8-import-order==0.18',
-            'pep8-naming==0.7.0'
+            'black==19.3b0'
```
Comment (wilzbach) [inline setup.py]: Why the beta release?

Comment (anukul) [reply setup.py]: https://github.com/psf/black#note-this-is-a-beta-product

File: Dockerfile [hunk 2]
```diff
@@ -1,18 +1,13 @@
...
 
 COPY          . /app/
-WORKDIR       /app
-RUN           chmod +x entrypoint.sh
 RUN           python setup.py install
-ENV           ASSET_DIR /asyncy
-ENV           logger_level info
 
-ENTRYPOINT    ["/app/entrypoint.sh"]
+ENTRYPOINT    ["storyscript-server", "start"]
```
Comment (wilzbach) [inline Dockerfile]: We should probably start testing the docker image to automatically ensure this didn't break anything.
See e.g. https://github.com/storyscript/runtime/issues/639

**Comment by wilzbach** [comment]
> Should we add a pre-commit hook that runs black?

Yep, though IIRC everyone needs to setup this individually in their respective git repository. You can only provide instructions and make the setup easier. It's still worth it to do this, because otherwise no one will try to set this up ;-)

**Pull request merged**